### ISSUE TITLE
"Toggle all checkboxes" control  is not accessible with keyboard:A11y_Azure Tools for VS Code_User can filter subscription_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-azurefunctions#4678]
Please do not close this bug. This bug should only be closed by Trusted Tester after verification.
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #Azure Tools for VS Code; #BM_AzureToolsforVSCode_Win32_AUG2025; #DesktopApp; #Win11; #WCAG2.1.1; #Keyboard; #FTP; #External; #External:VSCode; #FirstParty:Bug263645;

## Environment and OS details:
Visual Studio Code ( Version 1.103.1)
Azure function (Version 1.18.1 )
OS Version: Windows 11 Enterprise 24H2

## Repro Steps:

1. Open Visual studio code.
2. TAB till extension button and search for "Azure Functions". install it.
3. TAB till azure button and press NETR key
4. TAB till "Select subscription" button and press ENTER key.
5. TAB till "Toggle all checkboxes"
6. Observe the issue

## Actual Result:
When navigating using keyboard "Toggle all checkboxes" control  is not accessible.

## Expected Result:
"Toggle all checkboxes" control should be accessible with keyboard.

## User Impact:
Keyboard only user will unbale to access "Toggle all checkboxes" control .

## Attachment:

**Comment by MicroFish91** [comment]
Tracking upstream VS Code issue: https://github.com/microsoft/vscode/issues/263645

**Comment by kapilvaishna** [comment]
Issue is not repro in VS Code insider version 1.105.0 so closing the bug.

https://github.com/user-attachments/assets/ea9220a8-68ae-4f28-85f8-d7489bed485d
